Makefile

```diff
@@ -251,24 +251,21 @@ protobuf:
 		--mypy_out=lib \
 		proto/streamlit/proto/*.proto
 
-	@# Create a folder for autogenerated files
-	mkdir -p frontend/src/autogen
-
 	@# JS protobuf generation. The --es6 flag generates a proper es6 module.
 	cd frontend/ ; ( \
 		echo "/* eslint-disable */" ; \
 		echo ; \
 		yarn --silent pbjs \
 			../proto/streamlit/proto/*.proto \
 			-t static-module --wrap es6 \
-	) > ./src/autogen/proto.js
+	) > ./src/lib/proto.js
```

Comment:
We compile protobufs directly to the `lib` directory (no more `autogen` directory).